acasclient/selfile.py

```diff
@@ -0,0 +1,851 @@
+"""
+A module to read, edit, and write ACAS 'Single Experiment
+Loader' files.
+Common formats are supported for both input and output: csv, and
+excel
+"""
+# Contributors:  Shawn Watts
+
+
+################################################################################
+# Packages
+################################################################################
+import collections
+import logging
+import io
+import os
+from functools import partial
+import pandas as pd
+
+
+################################################################################
+# Globals/Constants
+################################################################################
+# File formats/types.
+CSV = '.csv'
+XLS = '.xls'
+XLSX = '.xlsx'
+LOT_CORP_NAME = 'Corporate Batch ID'
+GENERIC = 'Generic'
+DOSE_RESPONSE = 'Dose Response'
+
+# ACAS-CmpdReg constants in sdf files.
+PARENT_CORP_NAME = 'Parent Corp Name'
```

Comment:
I decided to leave these here for 2 reasons even though they aren't currently used within the acasclient package.

1. They could and should be used in the future but I didn't want to refactor other tests to use them right now.
2. By including them, the data team should be able to rely on pulling these from acasclient and remove their acas.py file